Bindings/Python/tests/test_access_subcomponents.py

```diff
@@ -13,11 +13,11 @@
 class TestAccessSubcomponents(unittest.TestCase):
     def test_individual_components(self):
         model = osim.Model(os.path.join(test_dir, "arm26.osim"))
-        muscle = model.getComponent('BICshort')
+        muscle = model.getComponent('forceset/BICshort')
         assert muscle.getName() == 'BICshort'
         # No downcasting necessary!
         muscle.get_max_isometric_force() # Method on Muscle.
-        muscle = model.updComponent('BICshort')
+        muscle = model.updComponent('forceset/BICshort')
```

Comment:
Is it true that "ForceSet" is a component now and contributes to component path but MuscleSet isn't (because it doesn't own) is the implication that all Force names need to be unique regardless of type?

Reply:
Yes, `ForceSet` is a Component and owns the `Force`s it contains. That ownership has not changed. Yes you cannot have two forces of the same name regardless of their subtype.